### ISSUE TITLE
Implement Accessibility Fixes following review

### ISSUE THREAD

**Issue by DeanWillicombe82** [ministryofjustice/data-platform#161]
### Summary

This epic enable us review/implement fixes to the Find MOJ Data front end following the internal review by the accessibility team. The review took place in March 2026 and was required after we found several accessibility issues during some internal testing. Due to changes in tech, personnel and the front end we felt a review was necessary. 

The fixes will ensure that all users of the service have the best experience and face no unexpected barriers. This will also ensure that when the product is merged with the Justice Data Platform we have a clear baseline. 

### Scope

This epic will cover all accessibility issues raised in the March 2026 review. 

### Goals/Desired Outcomes

We want out service to not only be 100% AA WCAG compliant but also most usable based on the feedback from the A-Team (Recommendations which go beyond WCAG)

### Dependencies & Constraints

There is an expectation that Content/Interaction Designers work together with developers to find the most effective solutions in each scenario. Where issues can't be resolved consideration should be given to including the exceptions to the Accessibility statement. 

We are also dependent on the design system components in many cases and fixes may need to be carried out by the relevant teams.

### Risks / Open Questions

Risks are that we introduce new issues or issues are not full resolved which could hamper user experience. 

Users could experience issues with the service during a period where we are dependent on design system teams making the necessary changes to components. 

### Links & References

[Link to issues document provided by A-Team](https://justiceuk.sharepoint.com/:x:/s/MoJAccessibilityTeam/IQAJ14tHjQWMRZTniFxmC5hMAavbuxOdwBiLu_TMi9ZBv14?email=Perla.Rembiszewski%40justice.gov.uk&e=7NtuyI)

[Link to Consultancy Notes](https://justiceuk.sharepoint.com/:w:/s/MoJAccessibilityTeam/IQCNMFERvfceQ5eAzqqwh1B-AZ3DdY9joo_CXCu8yKn7kpU?email=Perla.Rembiszewski%40justice.gov.uk&e=PpzunW)

[Link to Review playback recording on Teams](https://teams.microsoft.com/l/chat/19:meeting_ZTdiOWQyYTQtOTRiYy00Y2JmLWFmZTgtZjJmMTU2MTZkZGY2@thread.v2/conversations?context=%7B%22contextType%22%3A%22chat%22%7D)

### Epic-Level Success Criteria

- All tickets on this epic have been closed, whether implemented or not (indicating that all issues raised have been at least reviewed by the product team)

- Follow-up review by A-Team confirms that all issues are now resolved to a satisfactory manor.

**Comment by AdamDMOJ** [comment]
ID 2: [Issue #163](https://github.com/ministryofjustice/data-platform/issues/163)
ID 9: [Issue #227](https://github.com/ministryofjustice/data-platform/issues/227)
ID 10: [Issue #263](https://github.com/ministryofjustice/data-platform/issues/263)
ID 11: [Issue #273](https://github.com/ministryofjustice/data-platform/issues/273)
ID 12: Not implemented due to previous change from autofill to basic search.
ID 13: [Issue #377](https://github.com/ministryofjustice/data-platform/issues/377)
ID 14: Not Implemented due the user need to know if there are zero items within an asset type
ID 15: [Issue #258](https://github.com/ministryofjustice/data-platform/issues/258)
ID 16: [Issue #277](https://github.com/ministryofjustice/data-platform/issues/277) & [Issue #261](https://github.com/ministryofjustice/data-platform/issues/261)
ID 18: [Issue #276](https://github.com/ministryofjustice/data-platform/issues/276)
ID 19: [Issue #262](https://github.com/ministryofjustice/data-platform/issues/262)
ID 21: [Issue #467](https://github.com/ministryofjustice/data-platform/issues/467)
ID 22: [Issue #438](https://github.com/ministryofjustice/data-platform/issues/438)
ID 23: Glossary removed
ID 24: User guide will be removed in the future
ID 26: User guide will be removed in the future
ID 27: User guide will be removed in the future
ID 28: User guide will be removed in the future
ID 29: [Issue #336](https://github.com/ministryofjustice/data-platform/issues/336)
